textproc/libxmlxx5/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           meson 1.0
+
+
+# requires C++11 as of version 2.39.1
+compiler.cxx_standard 2017
+
+name                libxmlxx5
+set gname           libxml++
+version             5.0.1
+set branch          [join [lrange [split ${version} .] 0 1] .]
+categories          textproc
+license             LGPL
+maintainers         openmaintainer
```

Comment:
if you are not listing yourself as maintainer this should just be `nomaintainer`